_projects/2023/100481838/100481838.Rmd

```diff
@@ -0,0 +1,1067 @@
+---
+title: "We feel it already. How about we visualize it?"
+description: |
+  A short description of the post.
+categories: "2023"
+author: Mikaela De Smedt
+date: 2023-12-16
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+# Introduction
+
+Researchers have found that summer temperatures have shifted drastically since the mid-20th century. An eerie discovery that points to the frightening reality that temperatures in the warmer season of the year are now hot or extremely hot in comparison what they were 40 years ago.
+
+![Source: <https://www.nytimes.com/interactive/2021/climate/extreme-summer-heat.html>](Original%20chart.png)
+
+The aim of this tutorial is to analyze the visual encoding and replicate this chart. The final section also proposes additional improvements.
+
+## The plot in question: the story and how it's told
+
+The following chart uses data from retired NASA climate scientist James Hansen to show how summer temperatures have shifted toward more extreme heat over the past several decades.
+
+This representation uses bell curves to compare the distribution in the frequency of occurrence of temperature anomalies (y-axis) divided by local standard deviation for the 1951-1980 period (x-axis) obtained by binning all local results for the indicated region and 11-year period into 0.05 frequency intervals. Additionally, this chart is presented along an animation which shows the transition of all recorded periods.
+
+![Note: This is a screen recording of the actual animation. I was unable to properly download it due to the way it was embedded in the NYT website. Apologies for the resolution.](Untitled%20video%20-%20Made%20with%20Clipchamp.gif)
```

Comment:
This is nice for the presentation, but here the reader can go, click the link and see for themselves. Plus this video is too big. Please remove both the video and this reference to save some space in the repo. :)